documentation/asciidoc/accessories/camera/camera_hardware.adoc

```diff
@@ -2,6 +2,32 @@
 
 The Raspberry Pi Camera Modules are official products from the Raspberry Pi Foundation. The original 5-megapixel model was https://www.raspberrypi.org/blog/camera-board-available-for-sale/[released] in 2013, and an 8-megapixel https://www.raspberrypi.org/products/camera-module-v2/[Camera Module v2] was https://www.raspberrypi.org/blog/new-8-megapixel-camera-board-sale-25/[released] in 2016. For both iterations, there are visible light and infrared versions. A 12-megapixel https://www.raspberrypi.org/products/raspberry-pi-high-quality-camera/[High Quality Camera] was https://www.raspberrypi.org/blog/new-product-raspberry-pi-high-quality-camera-on-sale-now-at-50/[released] in 2020. There is no infrared version of the HQ Camera, however the xref:camera.adoc#raspberry-pi-hq-camera-filter-removal[IR Filter can be removed] if required.
 
+=== Installing a Raspberry Pi camera
```

Comment:
FYI once #2159 has been merged then "nested includes" will work properly, i.e. `camera/camera_hardware.adoc` will be able to include `camera_installing.adoc`, so there won't be any need to copy'n'paste the contents of adoc files into other adoc files. Letting you know in case that makes things any easier :wink: